pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -91,6 +99,10 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		{
 			ObjectMeta: kapi.ObjectMeta{
 				Name: ClusterAdminRoleName,
+				Annotations: map[string]string{
```

Comment:
instead of saying "granted to a user within a local policy" can you say "granted to a user on a project"